imsim/photon_ops.py

```diff
@@ -285,13 +253,14 @@ def deserialize_lsst_optics(config, base, _logger):
     shift_optics = opt_kwargs.pop("shift_optics", base.get("shift_optics", None))
 
     return LsstOptics(
-        telescope=base["_telescope"],
+        telescope=load_telescope_with_shift_optics(
+            kwargs["telescope"], kwargs["band"], shift_optics=shift_optics
+        ),
         boresight=kwargs["boresight"],
         sky_pos=base["sky_pos"],
         image_pos=base["image_pos"],
         icrf_to_field=base["_icrf_to_field"],
```

Comment:
This is also implicitly requiring the BatoidWCS.  Can we do the same trick for that as you did for the telescope?

Reply:
In my not-yet committed edits for adding bending mode perturbations, I rewrote how the batoid telescope instance was handled (there it's loaded by the config file "input" section processing).  I expect that will eventually supersede this too.

Reply:
@rmjarvis  This is not as easy as in the case of the telescope.
Right now, assembling `icrf_to_field` involves `BatoidWCSBuilder.makeWCSFactory`, which requires all the args necessary for `BatoidWCS`. Therefore refactoring this would require more time and I am not sure, how many of the args of `BatoidWCS` would be needed in the end.
But maybe the edits of @jmeyers314 will improve this a bit.

Reply:
Right.  Let's leave this as is for now, but hopefully we can make this a little more flexible with Josh's upcoming changes.